CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Streaming levels with query-based interest (QBI) enabled no longer produces errors if the player connection owns unreplicated actors.
 - Fixed an issue that would prevent player movement in a zoned deployment.
 - Fixed an issue that could cause queued incoming RPCs with unresolved references to never be processed.
+- Multicast RPCs are now processed after the EntityCreation request is satisfied.
```

Comment:
I'd phrase this as a bug fix as this is a little hard to parse.
Something like about how rpcs made soon after entity creation are now correctly processed on non owning clients.

Reply:
`Multicast RPCs, that are close to Actor creation request are not dropped now, but queued till the Actor creation request is satisfied.`

Reply:
Would that be good enough?

Reply:
It's still a bit misleading slightly odd grammar. Maybe:
`Muticast RPCs, that are sent shortly after after an actor is created, are now correctly processed by all clients.`
I don't think it matters that much though.

Reply:
Updated.